ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -37,37 +37,42 @@ gcp_connection::gcp_connection(const std::string &bucket_name)
 
 // Builds a list of object names from the bucket.
 int
-gcp_connection::list_objects(std::vector<std::string> &objects) const
+gcp_connection::list_objects(std::vector<std::string> &objects)
```

Comment:
let's place back the const for functions other than delete object

Reply:
fixed